ios/ViewController.mm

```diff
@@ -176,8 +179,8 @@ - (void)viewDidLoad {
 	}
 
 	g_dpi = (IS_IPAD() ? 200.0f : 150.0f) * scale;
-	g_dpi_scale_x = 240.0f / g_dpi;
-	g_dpi_scale_y = 240.0f / g_dpi;
+    g_dpi_scale_x = 1;//240.0f / g_dpi;
+    g_dpi_scale_y = 1;//240.0f / g_dpi;
```

Comment:
If possible, it'd be better to calculate `g_dpi` correctly and account for it in some way.  For example, if you connect this to a 6K display, the text may be unreadably small with scale of `1.0f`.

-[Unknown]

Reply:
It seems that iOS does not provide API to get physical size of any display (both internal and external). I implemented scaling based on diagonal length instead.